apps/site/assets/css/_customer-support.scss

```diff
@@ -258,9 +258,11 @@ body {
 }
 
 .ada-container {
+  display: flex;
```

Comment:
question: I'm not sure if there is a design file I should look at - is this the expected layout with the icon on small screens?
![image](https://user-images.githubusercontent.com/31781298/231280602-b5c685ea-f9b1-4afb-a4a5-d0b606499a76.png)

Reply:
@KaylaBrady we'll create a separate pr for this particular change